2/Step_five.rb

```diff
@@ -0,0 +1,4 @@
+# Заполнить массив числами от 10 до 100 с шагом 5
+
+fibonacci = Array.new
```

Comment:
```suggestion
fibonacci = []
```
В руби в 99% случаев используют литерал для создания пустого массива. А 1% использует `Array.new`, потому что им еще не сказали про литералы